src/tests/system/tests/test_passkey.py

```diff
@@ -241,3 +241,188 @@ def test_passkey__su_srv_not_resolvable(
         ioctl=f"{moduledatadir}/umockdev.ioctl",
         script=f"{testdatadir}/umockdev.script.{suffix}",
     )
+
+
+@pytest.mark.importance("high")
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+@pytest.mark.builtwith(client="passkey", provider="passkey")
+def test_passkey__offline_su(client: Client, provider: GenericProvider, moduledatadir: str, testdatadir: str):
+    """
+    :title: Check offline su authentication of a user with LDAP, IPA, AD and Samba
+    :setup:
+        1. Add a LDAP, IPA, AD and Samba user with passkey_mapping.
+        2. In SSSD domain change "cache_credentials" and "krb5_store_password_if_offline" to "True"
+        3. In SSSD pam change "offline_credentials_expiration" to "0"
+        4. Setup SSSD client with FIDO and umockdev, start SSSD service.
+    :steps:
+        1. Check su authentication of the user.
+        2. Make server offline by removing rules for LDAP, KDC and Global Catalog ports.
+        3. Bring SSSD offline explicitly.
+        4. Check su offline authentication of the user.
+    :expectedresults:
+        1. User su authenticated successfully.
+        2. Firewall rules dropped.
+        3. SSSD is offline.
+        4. Offline su authentication is successful.
+    :customerscenario: False
+    """
+    suffix = type(provider).__name__.lower()
+
+    with open(f"{testdatadir}/passkey-mapping.{suffix}") as f:
+        provider.user("user1").add().passkey_add(f.read().strip())
+
+    if suffix == "ldap":
+        client.sssd.domain["local_auth_policy"] = "only"
+
+    client.sssd.domain["cache_credentials"] = "True"
+    client.sssd.domain["krb5_store_password_if_offline"] = "True"
```

Comment:
This is only valid for 2FA, and from the SSSD perspective this isn't 2FA. The pin (or fingerprint) that you are setting is handled by FIDO2. Therefore, it is not necessary to set this value.

Reply:
updated.